.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           name: Setup test environment
           command: |
             sudo apt-get update
-            sudo apt-get install -y imagemagick icc-profiles-free
+            sudo apt-get install -y imagemagick icc-profiles-free ghostscript
```

Comment:
Minor: Should we update this dep in the `README.md` note too?

Reply:
yes that's important for the manual test runs. Updated.